src/plugins/simpleini/simpleini.c

```diff
@@ -126,8 +249,17 @@ int elektraSimpleiniGet (Plugin * handle, KeySet * returned, Key * parentKey)
 	}
 
 	char * key = 0;
+	char * strippedkey = 0;
 	char * value = 0;
 	int errnosave = errno;
+	
+	char * format = getReadFormat (handle);
+	if (!format)
+	{
+		ELEKTRA_SET_ERROR (ELEKTRA_ERROR_NOEOF, parentKey, "invalid 'format' specified");
```

Comment:
Ideally it would already be validated by `checkConf`, see https://master.libelektra.org/doc/tutorials/plugins.md which is, however, currently not present in simpleini.